src/node_binding.cc

```diff
@@ -190,6 +190,52 @@ void InitModpendingOnce() {
   CHECK_EQ(0, uv_key_create(&thread_local_modpending));
 }
 
+#define ABI_VERSION_CASE(entry, key, KEY)                                     \
+  case node_abi_ ## key ## _version:                                          \
+    if ((entry)->version != NODE_ABI_ ## KEY ## _VERSION) {                   \
+      char int_str[32];                                                       \
+      result += std::string("\n") +                                           \
+        NODE_STRINGIFY(node_abi_ ## key ## _version) +                        \
+        ": module: ";                                                         \
+      snprintf(int_str, sizeof(int_str), "%d", (entry)->version);             \
+      result += int_str;                                                      \
```

Comment:
You should be able to use `std::to_string((entry)->version)` instead of `sprintf`, btw.

Reply:
@addaleax I've had bad experiences with `std::to_string` – like it not being available on the OSX STL even though it was claiming to be C++11. I'll use it and see what the CI says.